packages/blocks/src/api/raw-handling/paste-handler.js

```diff
@@ -0,0 +1,240 @@
+/**
```

Comment:
Why is moving to a separate file needed?

Reply:
👋 @iseulde , during a Slack chat with @mkevins , I suggested that in order to limit scope, we should extract only the method we currently need on mobile, which is the `pasteHandler()` method. That's then exported by the `index.native.js` module.

Reply:
It's then used in `rich-text/index.native.js`.

Reply:
Ok. I see any the moving of web code but... nothing else has changed. The extra checks look ok.

Reply:
I understand you completed your review pass @iseulde . Thanks for the quick response! 🙇‍♂️

Reply:
An `index.native.js` file already existed for `raw-handling`, and I was not able to import the vanilla `pasteHandler` from `index.js`.

Reply:
Thank you @iseulde for having a look at my proposed changes, and for your feedback.